src/main/resources/kits.yml

```diff
@@ -56,4 +56,18 @@ Kit-Cooldown:
   Wizard:
     Essence: 15
     Staff: 1
-  Zombie-Finder: 30
\ No newline at end of file
+  Zombie-Finder: 30
+
+# The settings for some kits
+Kit-Settings:
+  Cleaner:
+    # The maximum health for the enemies to be removed
+    Max-Health: 2048
+    # The maximum amount of enemies that this kit can remove
+    Max-Amount: 50
+    # From the start, how many enemies can this kit remove?
+    Base-Amount: 10
+    # The amount of the removed enemies will be increased after how many waves?
+    Increase-After-Wave: 5
+    # How many amount will be added to the base amount
```

Comment:
You may document that a value of zero (0) does not increase the amount of removal.